src/utils/CourseChecklist/courseChecklistData.jsx

```diff
@@ -0,0 +1,77 @@
+export const filters = {
```

Comment:
Nit: Not necessary for this PR since we're going to be looking at this still, but this should probably have a slightly more descriptive name.